testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -89,10 +91,11 @@ protected QueryRunner createQueryRunner()
             public Iterable<ConnectorFactory> getConnectorFactories()
             {
                 MockConnectorFactory connectorFactory = MockConnectorFactory.builder()
-                        .withListTables((session, s) -> ImmutableList.of(new SchemaTableName("default", "test_table")))
+                        .withListTables((session, s) -> ImmutableList.of(new SchemaTableName("default", "tests_table")))
```

Comment:
undo? why these changes? seem unrelated

Reply:
They are not unrelated, `TestEventListenerBasic#testPlanningFailure` starts failing after the first PushProjectionIntoTableScan is moved.

Reply:
We do we need to change `test_table` to `tests_table`?
Why do we need `getColumns`?

Reply:
`test_table` is wrong as the query in test uses `tests_table `
We need some non-trivial projection on an actual column now to avoid the ProjectNode from getting pruned before PushProjectionIntoTableScan has a chance to execute.